test/parallel/test-fs-realpath.js

```diff
@@ -201,9 +201,11 @@ function test_cyclic_link_protection(callback) {
     fs.symlinkSync(t[1], t[0], 'dir');
     unlink.push(t[0]);
   });
-  assert.throws(function() { fs.realpathSync(entry); });
+  assert.throws(function() { fs.realpathSync(entry); }, /^Error: ELOOP/);
```

Comment:
>Another thing that might help is if the arguments accepted by common.expectsError() was a single settings object. 

Yeah that looks way easier to understand (as someone who hasn't yet gotten round to reading the source or docs for `common.expectsError()`).